### ISSUE TITLE
(wav2vec2) Regularisation parameters do not match those in the paper

### ISSUE THREAD

**Issue by RuABraun** [pytorch/fairseq#3492]
## ❓ Questions and Help

If I look at `examples/wav2vec/config/finetuning/base_10h.yaml` I see the following settings
```
  apply_mask: true
  mask_prob: 0.65
  mask_channel_prob: 0.5
  mask_channel_length: 64
  layerdrop: 0.05
  activation_dropout: 0.1
  feature_grad_mult: 0.0
  freeze_finetune_updates: 10000
```
But in Appendix B of the [paper](https://arxiv.org/pdf/2006.11477.pdf) I see that `mask_channel_prob` was apparently set to `0.008` for the 10h finetuning. Why the big difference?

Also, in the paper the `timestep mask prob.` is mentioned. I cannot find any reference to this in the repository.

**Comment by alexeib** [comment]
they just use different units. in the paper we tried to make it simpler and present the prob mask as probability for each individual timestep to start a masking span. but in the code, we actually use a parameter that should be interpreted as approximately what percentage of total number of timesteps to mask. we compute the number of timesteps to sample as starting states as mask_prob * sample_length / mask_span_length.

mask_channel_prob is what corresponds to timestep mask prob

**Comment by RuABraun** [comment]
Thank you for explaining!
